### PR TITLE
[WNMGDS-2365] Move CSS reset migration guide to a permanent place

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

- Move CSS reset migration guide to a permanent place under "Foundations"
- I also removed the spacing conversation from it. Spacing change is a delta, but the reset (normalization) is a core concept moving forward. This content will be moved into the release's blog post most likely.

### CODE REVIEW

File: packages/docs/content/foundation/css-normalization.mdx [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 
-If you're curious to see what changes have been made, you can preview our reset styles in [Storybook](/storybook/?path=/story/reset-overview--overview). We'll break down the major changes for you to review below.
+<Alert heading="Why do we need to normalize CSS?">
+Each browser has default styles that it applies to various HTML elements. Not all browser have the same default styles (called user-agent styles), which means that websites without custom CSS will look different when viewed from different browsers. Even when creating custom CSS, web developers must take those different default styles into account and make sure they are overridden appropriately. This can be tedious and error-prone, so a common practice among developers is to either remove or normalize (remove the differences in) those user-agent styles with a collection of base CSS rules before applying any component CSS that controls the look-and-feel of the site.
```
Comment (zarahzachz) [inline packages/docs/content/foundation/css-normalization.mdx:13]: Typo:
- "Not all browser have the same" should be "Not all browsers have the same"

**Review by zarahzachz — APPROVED** [review]
Saw one typo but otherwise looks good

**Review by forrestbaer — APPROVED** [review]
I like it! 'Normalization' rather than Reset is a good call.

File: packages/docs/content/foundation/css-normalization.mdx (hunk 1 above)
Comment (pwolfert) [reply packages/docs/content/foundation/css-normalization.mdx:13]: Thanks!

**Pull request merged**